packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +86,34 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+///An extension of the expansion panel class that allows for radio functionality.
```

Comment:
"An extension of the expansion panel class" is an implementation detail, instead talk about the radio functionality:

"An expansion panel which ..."